src/main/java/tasks/Task.java

```diff
@@ -0,0 +1,49 @@
+package tasks;
+
+public class Task {
```

Comment:
Would it be better if we can turn this class to an abstract class, because in real application, we don't actually instantiate a "general" task?